doc/source/docs/user_guide/aggregation_with_dissolve.rst

```diff
@@ -75,3 +75,23 @@ However it also accepts other summary statistic options as allowed by :meth:`pan
 * 'sum'
 * 'mean'
 * 'median'
+* function
+* string function name
+* list of functions and/or function names, e.g. [np.sum, 'mean']
+* dict of axis labels -> functions, function names or list of such.
+
+For example, to get the number of contries on each continent, 
+as well as the populations of the largest and smallest country of each,
+we can aggregate the ``'name'`` column using ``'count'``,
+and the ``'pop_est'`` column using ``'min'`` and ``'max'``:
+
+.. ipython:: python
+
+   world = geopandas.read_file(geopandas.datasets.get_path('naturalearth_lowres'))
+   continents = world.dissolve(by='continent', aggfunc=
+   {
+      'name': 'count',
+      'pop_est': ['min', 'max'],
+   })
```

Comment:
```suggestion
    world = geopandas.read_file(geopandas.datasets.get_path("naturalearth_lowres"))
    continents = world.dissolve(
        by="continent",
        aggfunc={
            "name": "count",
            "pop_est": ["min", "max"],
        },
    )
```

I've just applied `black` formatting here for consistency and better readability.

Reply:
Thanks!
(BTW, is there a way to invoke black on code within RST files?)